comments-by-guideline-and-success-criterion.md

```diff
@@ -249,12 +249,27 @@ With these substitutions, this success criterion would read:
 
 This applies directly as written, and as described in [Intent from Understanding Success Criterion 2.4.6](http://www.w3.org/WAI/WCAG22/Understanding/headings-and-labels#intent) (also provided below).
 
-<div class="note">In [software](http://w3c.github.io/wcag2ict/#wcag2ict-def_software), headings and labels are used to describe sections of [content](http://w3c.github.io/wcag2ict/#wcag2ict-def_content) and controls respectively. In some cases it may be unclear whether a piece of static text is a heading or a label. But whether treated as a label or a heading, the requirement is the same: that if they are present they describe the topic or purpose of the item(s) they are associated with.</div>
 
 ##### focus-visible
 
 This applies directly as written, and as described in [Intent from Understanding Success Criterion 2.4.7](http://www.w3.org/WAI/WCAG22/Understanding/focus-visible#intent) (also provided below).
 
+<p class="ednote">This section is to be developed by the WCAG2ICT Task Force.</p>
```

Comment:
Focus visible is already present in the document and only changed from Level AA to Level A. So I don't think we'll have to change anything.